howto/build-env/README.md

```diff
@@ -37,15 +37,15 @@ errors to allow the normal deprecation process to work:
 
 ### Xcode
 
-The core team uses **Xcode 8 and Swift 3.0**
+The core team tests against **Xcode 8.3.3, Xcode 9.0.1, Xcode 9.1 and Swift 3.2**
```

Comment:
We should probably say something about trying to support Swift 4 whenever possible. We do get PRs that are meant to assist the experience.

Reply:
I think you can edit this PR if you'd like to add the additional language.